plenario/static/js/explore/templates/aboutTemplate.html

```diff
@@ -16,7 +16,7 @@
           <a href="javascript://" class='about-detail' data-dataset_name="<%= set.dataset_name %>"><%= set.human_name %></a>
         </td>
         <td>
-          <% if (set.obs_from && set.obs_to) { %>
+          <% if (set.obs_from && set.obs_to && moment(set.obs_to).isBefore(moment()) && moment(set.obs_from).isAfter('1989-12-31')) { %>
```

Comment:
Two concerns here:
1) I think making sure `obs_from` is > 1989 is a bit too restrictive. That cuts off, say, the Chicago historic landmarks dataset.
2) In general, I'm uncomfortable about literal values like `1989-12-31`. When you insert literal values like this, at the least be sure to include a comment justifying your choice.